python/python/tests/test_transaction.py

```diff
@@ -129,6 +130,15 @@ async def transaction_test(
     transaction.hdel(key4, [key, key2])
     args.append(2)
 
+    transaction.hset(key11, {key: value})
```

Comment:
you can still use key4, from what I see it contains one field which is {key3: "10.5"}